src/components/Loading/Loading.jsx

```diff
@@ -1,7 +1,18 @@
+import { useState, useEffect } from "react"
+
+
 const Loading = () => {
 
+    const [message, setMessage] = useState("Page Is Loading...")
+
+    useEffect(() => {
+        const timeout = setTimeout(() =>{
+            setMessage("Page is still loading...  This is most likely due to the server reinitialising after a period of inactivity.  Thank you for your patience")
```

Comment:
We're setting this message 5 seconds after the loading component has been rendered? it seems a long time to wait for this message!

Reply:
Love the idea though! Great for UX